.gitignore

```diff
@@ -3,6 +3,10 @@
 # C extensions
 *.so
 
+# vim temporal files
```

Comment:
No worries, glad to help! Thanks for working with me in my push for perfection 👍